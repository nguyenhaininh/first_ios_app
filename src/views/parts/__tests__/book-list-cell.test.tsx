/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import BookListCell from '../book-list-cell';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <BookListCell
      title="Book Title"
      subTitle="Sub Title"
      new={true}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
