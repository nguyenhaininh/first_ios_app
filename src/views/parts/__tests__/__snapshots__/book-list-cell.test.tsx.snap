// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={Object {}}
>
  <View />
  <View
    style={Object {}}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#000000",
          "fontSize": 12,
          "fontWeight": "bold",
          "marginTop": 10,
        }
      }
    >
      Book Title
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginTop": 6,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#999999",
            "fontSize": 11,
          }
        }
      >
        Sub Title
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "borderColor": "#e62222",
            "borderWidth": 1,
            "height": 12,
            "justifyContent": "center",
            "width": 27,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#e62222",
              "fontSize": 8,
            }
          }
        >
          NEW
        </Text>
      </View>
    </View>
  </View>
</View>
`;
